internal/policies/mount/mount.go

```diff
@@ -96,38 +121,27 @@ func (m *Manager) ApplyPolicy(ctx context.Context, objectName string, isComputer
 	defer m.mountsMu[objectName].Unlock()
 
 	if len(entries) == 0 {
-		if err := m.cleanup(ctx, objectName, isComputer); err != nil {
-			return err
-		}
-		return nil
+		return m.cleanup(ctx, objectName, isComputer)
 	}
 
-	if !isComputer {
-		for _, entry := range entries {
-			switch entry.Key {
-			case "user-mounts":
-				if e := m.applyUserMountsPolicy(ctx, objectName, entry); e != nil {
-					err = e
-				}
-
-			default:
-				log.Debugf(ctx, "Key %q is currently not supported by the mount manager", entry.Key)
-			}
-		}
-	} else {
-		for _, entry := range entries {
-			switch entry.Key {
-			case "system-mounts":
-				if e := m.applySystemMountsPolicy(ctx, objectName, entry); e != nil {
-					err = e
-				}
-			default:
-				log.Debugf(ctx, "Key %q is currently not supported by the mount manager", entry.Key)
-			}
-		}
+	key := "user"
+	if isComputer {
+		key = "system"
+	}
+
+	i := slices.IndexFunc(entries, func(e entry.Entry) bool {
+		return e.Key == key+"-mounts"
+	})
+
+	if i == -1 || entries[i].Disabled {
+		log.Debugf(ctx, i18n.G("The provided entries are not supported by the %s mount manager: %v"), key, entries)
```

Comment:
Is this covered by test? (I didn’t run coverage but I haven’t seen any tests that could cover it)

Also, a disabled key is a valid use case, no? We just skip it but we shouldn’t log it as unsupported

Finally, this will require more changes if one day we supports more keys, but this is ok I guess.